tex/introduction.tex

```diff
@@ -1,114 +1,82 @@
+MayergoyzFredkinZhang2005, MayergoyzZhang2007}, interactions between nanoparticles
+and electron beams \cite{GarciadeabajoAizpurua1997, GarciadeabajoHowie2002},
+and surface plasmon resonance sensors.
+Among the latter application we find biosensors, where researchers have modelled the 
+interaction between the metallic nanoparticle and target molecules with highly 
+simplified methods \cite{JungCampbellChinowskyMarYee1998,HaesVanduyne2002,DavisGomezVernon2010,AntosiewiczApellClaudioKall2011}.
+
+
+{\color{red} Not sure how to talk about the different methods, any idea? should
+we talk about this? 
+
+Methods: fem (comsol), fdtd (davis and coworker), bem (garcia de abajo and 
+howie, bem ++, matlab guy). van duyne and haes? }
+{\color{blue} wrote something there, let me know what you think. My impression is that
+we shouldnt talk too much on this. We do cite everybody in the 'applications' part of the paragraph, so I dont think we nee to cite them also in the 'methods' part}
 
 \begin{figure}[h] %  figure placement: here, top, bottom, or page
    \centering
    \includegraphics[width=0.35\textwidth]{lspr.pdf} 
-   \caption{Localized Surface Plasmon Resonance (LSPR) scheme. LSPR is an 
-            optical phenomenon that ocurrs when light shines on conductive 
-            nanoparticles that are smaller than the wavelength of the incident 
-            light. The free electrons on the surface of the nanoparticle are 
-            excited by the incoming electric field oscillating with it and 
-            creating plasmons}
+   \caption{Localized Surface Plasmon Resonance (LSPR) scheme. }
    \label{fig:lspr}
 \end{figure}
 
+%* What is unkown, limitations and gaps
 
-{\color{red} Not sure about where you are taking this introduction.
-Perhaps we should discuss a little bit on the point of view of the paper before anything.
-We need to be VERY careful about the introduction because it sets the tone, and allows us to highlight what we think is important, but this has to be clear among all authors. 
-
-My impression is that the key result of the paper is that it is the first time that a detailed molecular structure is placed next to the nanoparticle.
-The whole discussion on free software, reproducibility, and convergence analysis is interesting, but I don't feel it is central of this paper. 
-We've discussed this several times, and even though we know that these topics are extremely important, the rest of the community doesn't necessarily feel that way.
-I'm afraid making this the central discussion of the paper might put off editors. 
-I would keep the paper more technical.
-Plus, we need to be careful not to be looking for everybody else's limitations, rather than focusing on our work.
+In the biosensor community most progress is achieved 
+through experimental analysis, with trial and error procedures {\color{blue} I also have this impression, but do we have a reference to support it? If not, I'd rather avoid it}. 
```

Comment:
@cdcooper84  This is one of the latest reviews I found on LSPR, where most of the work cited here looks experimental. It's a review from 2015, take a look and tell me if it's enough. If not we can remove this statement. 
https://www.mdpi.com/1424-8220/15/7/15684/htm

Reply:
I see your point, however, that paper does cite some (few) simulations. I just want to avoid any problems if we get a difficult referee. 

Reading it again, it doesn't sound so bad. Let's keep it in the meantime, perhaps Lorena has another view of this.